chapter_12_organising_test_files.asciidoc

```diff
@@ -378,14 +381,15 @@ AssertionError: write me!
 Brilliant--no need to sit around waiting for all the FTs
 when we're only interested in a single one.
 Although we need to remember to run all of them now and again, to check for regressions.
+It's a good habit to run all the tests before making a `git commit`.
 Later in the book we'll see how to give that task over to an automated Continuous Integration loop.
 For now, let's commit!
```

Comment:
i'm worried this ends up being ambiguous - are we giving the task of making commits over to CI?

```suggestion
Although we need to remember to run all of them now and again, to check for regressions.
Later in the book we'll set up a Continuous Integration (CI) server to run all the tests automatically,
for example every time we push to master.
For now, a good prompt for running all the tests is "just before you do a commit",
so let's get into that habit now:
```

Reply:
Nice, I like it!

Reply:
maybe we'd want to mark it as xfail then?